addons/mail/static/src/components/discuss/discuss.xml

```diff
@@ -16,6 +16,9 @@
                     <DiscussSidebar class="o_Discuss_sidebar"/>
                 </t>
                 <t t-if="env.messaging.device.isMobile" t-call="mail.Discuss.content"/>
+                <t t-if="env.messaging.chatRoomLocalId">
+                    <VideoRoom roomLocalId="env.messaging.chatRoomLocalId"/>
+                </t>
```

Comment:
Confusing naming between VideoRomm and chatRoom, but both should just be part of channel.

So I guess `currentlyActiveRtcChannel` or something like that should be the field.
Note: we don't ever reference localId in field names, only records.
Only props use localId and it's a technical limitation, not a choice.